airbyte-integrations/connectors/source-postgres/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/PostgresSourceStrictEncryptAcceptanceTest.java

```diff
@@ -44,11 +47,15 @@ public class PostgresSourceStrictEncryptAcceptanceTest extends SourceAcceptanceT
   private PostgreSQLContainer<?> container;
   private JsonNode config;
 
+  protected static final String PASSWORD = "Passw0rd";
+  protected static Triple<String, String, String> certs;
+
   @Override
   protected void setupEnvironment(final TestDestinationEnv environment) throws Exception {
-    container = new PostgreSQLContainer<>(DockerImageName.parse("marcosmarxm/postgres-ssl:dev").asCompatibleSubstituteFor("postgres"))
-        .withCommand("postgres -c ssl=on -c ssl_cert_file=/var/lib/postgresql/server.crt -c ssl_key_file=/var/lib/postgresql/server.key");
```

Comment:
If you mean everything marked in the `withCommand` parameter, then yes, this functionality has moved to the `getCertificate()` method. Also, I'm using the official postgres docker container, instead of the custom one that marcosmarxm created.